pulsar-broker/src/main/java/com/yahoo/pulsar/broker/service/Consumer.java

```diff
@@ -546,5 +553,15 @@ public Subscription getSubscription() {
         return subscription;
     }
     
+    private int addAndGetUnAckedMsgs(Consumer consumer, int ackedMessages) {
```

Comment:
@rdhabalia Why did you call this `ackedMessages` while you're adding (`addUnackedMessages`) - it's very confusing. I had to search through the code to find you added `-` to it 2 calls up the stack. Very time consuming.